lms/envs/common.py

```diff
@@ -1248,12 +1248,34 @@
 base_application_js = [
```

Comment:
Note to reviewers: base_application_js is used instead of application_js if you specify disable_courseware_js. See https://github.com/edx/edx-platform/blob/master/lms/templates/main.html#L67.

We are trying to avoid downloading module-js (the common xmodule code), which is huge.